spec/active_record_cloning_spec.rb

```diff
@@ -0,0 +1,95 @@
+require 'spec_helper'
+
+connect_to_sqlite
+
+describe ActiveRecordCloning do
+  let(:attrs) {
+    {
+      name: 'Bill',
+      age: '50',
+      position: 6,
+      category_id: 1
+    }
+  }
+  subject(:record) { model_class.create!(attrs) }
```

Comment:
this is never used as a subject for a test, can be a simple let to avoid confusion